charts/vald/templates/_helpers.tpl

```diff
@@ -456,6 +456,7 @@ addrs: []
 health_check_duration: {{ default .default.health_check_duration .Values.health_check_duration | quote }}
 connection_pool:
   {{- if .Values.connection_pool }}
+  resolve_dns: {{ default .default.connection_pool.resolve_dns .Values.connection_pool.resolve_dns }}
```

Comment:
I think bool parameter should have "enable" or "disable" or "is" prefix.

Reply:
I think about it but, it's difficult to understand for user because if some params start with enable
`enable-a-feature: disable` means disable a feature,
`disable-a-feature:  disable` means disable or not?
we have to think about it

but this time I use verb for naming `resolve DNS ? T/F` I think it's more clear 

@kevindiu what do you think